packages/story-editor/src/components/library/panes/media/local/mediaEditDialog.js

```diff
@@ -182,13 +187,24 @@ function MediaEditDialog({ resource, onClose }) {
               height
             )}
           </Text>
-          <AssistiveTextInput
-            value={altText}
-            aria-label={isImage ? imageInputTitle : videoInputTitle}
-            type="text"
-            placeholder={isImage ? imageInputTitle : videoInputTitle}
-            onChange={handleAltTextChange}
-          />
+
+          {isImage ? (
+            <AssistiveTextInput
+              value={altText}
+              aria-label={imageInputTitle}
+              type="text"
+              placeholder={imageInputTitle}
+              onChange={handleAltTextChange}
+            />
+          ) : (
+            <AssistiveTextArea
```

Comment:
Any downside of just always using a textarea?

Reply:
Not really. Will update since that will make it a bit cleaner.